web/scripts/copy-well-known.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2025 The Ground Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { existsSync, mkdirSync, readFileSync, writeFileSync } from 'fs';
+
+const project = process.env['npm_config_project'];
```

Comment:
Please add a JSDoc comment explaining what this file does and why it's needed.

Reply:
"well-known" should also be qualified (well-known what?), since without context it has several interpretations.

Reply:
ok, I renamed it into asset-links. more readable and correct while .well-known is the folder where firebase automatically looks for it by default.